src/index.ts

```diff
@@ -2,9 +2,28 @@
 import { Command } from 'commander';
 import path from 'path';
 
-import { generateChakra, generateJson } from './generate';
+import { generateChakra, generateJson, generateCss } from './generate';
 import { version } from '../package.json';
 
+const OPTIONS = {
```

Comment:
This feels a bit clunky to me, and is likely to get clunkier as we add more things. 
Is there a more elegant way of sharing common options etc across commands in `Commander`? 
or would even reorganising it into something like `figma2theme generate --type chakra` rather than a seperate command for each make sense?

Reply:
ooo that's a really good question, I like the sound of `figma2theme generate --type chakra`
@spykr any thoughts on this?

Reply:
Having one `generate` command instead of multiple sounds like a good idea 👍 I like `figma2theme generate chakra`, `figma2theme generate json`, `figma2theme generate css --latest-changes`, etc.

We would have to keep in mind that this wouldn't be backwards compatible so if any existing projects updated `figma2theme` their existing docs and `package.json` commands would become incorrect.